package.json

```diff
@@ -194,6 +194,10 @@
     "eslint-plugin-jsx-a11y": "6.7.1",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "4.6.0",
+    "grunt": "^1.6.1",
+    "grunt-cli": "^1.4.3",
+    "grunt-contrib-concat": "^2.1.0",
```

Comment:
Been a while since I've toiled w/ Grunt, but do we also need to include grunt-contrib-copy?
https://github.com/mozilla/fxa/blob/e2d29887248b5ed1daaf60616992e873f0790bed/apps/payments/next/Gruntfile.js#L41

Oh, I think we do; or at least we did for fxa-auth-server and fxa-content-server packages:
```sh
git grep "grunt-contrib-copy" -- packages/**/package.json | cat

packages/fxa-auth-server/package.json:    "grunt-contrib-copy": "1.0.0",
packages/fxa-content-server/package.json:    "grunt-contrib-copy": "1.0.0",
```

Reply:
Ah good catch. Added.